### ISSUE TITLE
gradlew not working 

### ISSUE THREAD

**Issue by justgo97** [s1lentq/ReGameDLL_CS#149]
```
C:\2017\ReGameDLL_CS-master>gradlew --max-workers=1 clean buildFixes --stacktrac
e
:buildSrc:compileJava UP-TO-DATE
:buildSrc:compileGroovy UP-TO-DATE
:buildSrc:processResources UP-TO-DATE
:buildSrc:classes UP-TO-DATE
:buildSrc:jar UP-TO-DATE
:buildSrc:assemble UP-TO-DATE
:buildSrc:compileTestJava UP-TO-DATE
:buildSrc:compileTestGroovy UP-TO-DATE
:buildSrc:processTestResources UP-TO-DATE
:buildSrc:testClasses UP-TO-DATE
:buildSrc:test UP-TO-DATE
:buildSrc:check UP-TO-DATE
:buildSrc:build UP-TO-DATE

FAILURE: Build failed with an exception.

* Where:
Build file 'C:\2017\ReGameDLL_CS-master\build.gradle' line: 19

* What went wrong:
A problem occurred evaluating root project 'regamedll'.
> Cannot get property 'tag' on null object

* Try:
Run with --info or --debug option to get more log output.

* Exception is:
org.gradle.api.GradleScriptException: A problem occurred evaluating root project
 'regamedll'.
        at org.gradle.groovy.scripts.internal.DefaultScriptRunnerFactory$ScriptR
unnerImpl.run(DefaultScriptRunnerFactory.java:76)
        at org.gradle.configuration.DefaultScriptPluginFactory$ScriptPluginImpl$
1.run(DefaultScriptPluginFactory.java:148)
        at org.gradle.configuration.DefaultScriptPluginFactory$ScriptPluginImpl.
apply(DefaultScriptPluginFactory.java:156)
        at org.gradle.configuration.project.BuildScriptProcessor.execute(BuildSc
riptProcessor.java:39)
        at org.gradle.configuration.project.BuildScriptProcessor.execute(BuildSc
riptProcessor.java:26)
        at org.gradle.configuration.project.ConfigureActionsProjectEvaluator.eva
luate(ConfigureActionsProjectEvaluator.java:34)
        at org.gradle.configuration.project.LifecycleProjectEvaluator.evaluate(L
ifecycleProjectEvaluator.java:55)
        at org.gradle.api.internal.project.AbstractProject.evaluate(AbstractProj
ect.java:487)
        at org.gradle.api.internal.project.AbstractProject.evaluate(AbstractProj
ect.java:85)
        at org.gradle.execution.TaskPathProjectEvaluator.configureHierarchy(Task
PathProjectEvaluator.java:42)
        at org.gradle.configuration.DefaultBuildConfigurer.configure(DefaultBuil
dConfigurer.java:35)
        at org.gradle.initialization.DefaultGradleLauncher.doBuildStages(Default
GradleLauncher.java:129)
        at org.gradle.initialization.DefaultGradleLauncher.doBuild(DefaultGradle
Launcher.java:106)
        at org.gradle.initialization.DefaultGradleLauncher.run(DefaultGradleLaun
cher.java:86)
        at org.gradle.launcher.exec.InProcessBuildActionExecuter$DefaultBuildCon
troller.run(InProcessBuildActionExecuter.java:90)
        at org.gradle.tooling.internal.provider.ExecuteBuildActionRunner.run(Exe
cuteBuildActionRunner.java:28)
        at org.gradle.launcher.exec.ChainingBuildActionRunner.run(ChainingBuildA
ctionRunner.java:35)
        at org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProce
ssBuildActionExecuter.java:41)
        at org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProce
ssBuildActionExecuter.java:28)
        at org.gradle.launcher.exec.DaemonUsageSuggestingBuildActionExecuter.exe
cute(DaemonUsageSuggestingBuildActionExecuter.java:50)
        at org.gradle.launcher.exec.DaemonUsageSuggestingBuildActionExecuter.exe
cute(DaemonUsageSuggestingBuildActionExecuter.java:27)
        at org.gradle.launcher.cli.RunBuildAction.run(RunBuildAction.java:40)
        at org.gradle.internal.Actions$RunnableActionAdapter.execute(Actions.jav
a:169)
        at org.gradle.launcher.cli.CommandLineActionFactory$ParseAndBuildAction.
execute(CommandLineActionFactory.java:237)
        at org.gradle.launcher.cli.CommandLineActionFactory$ParseAndBuildAction.
execute(CommandLineActionFactory.java:210)
        at org.gradle.launcher.cli.JavaRuntimeValidationAction.execute(JavaRunti
meValidationAction.java:35)
        at org.gradle.launcher.cli.JavaRuntimeValidationAction.execute(JavaRunti
meValidationAction.java:24)
        at org.gradle.launcher.cli.CommandLineActionFactory$WithLogging.execute(
CommandLineActionFactory.java:206)
        at org.gradle.launcher.cli.CommandLineActionFactory$WithLogging.execute(
CommandLineActionFactory.java:169)
        at org.gradle.launcher.cli.ExceptionReportingAction.execute(ExceptionRep
ortingAction.java:33)
        at org.gradle.launcher.cli.ExceptionReportingAction.execute(ExceptionRep
ortingAction.java:22)
        at org.gradle.launcher.Main.doAction(Main.java:33)
        at org.gradle.launcher.bootstrap.EntryPoint.run(EntryPoint.java:45)
        at org.gradle.launcher.bootstrap.ProcessBootstrap.runNoExit(ProcessBoots
trap.java:54)
        at org.gradle.launcher.bootstrap.ProcessBootstrap.run(ProcessBootstrap.j
ava:35)
        at org.gradle.launcher.GradleMain.main(GradleMain.java:23)
        at org.gradle.wrapper.BootstrapMainStarter.start(BootstrapMainStarter.ja
va:30)
        at org.gradle.wrapper.WrapperExecutor.execute(WrapperExecutor.java:127)
        at org.gradle.wrapper.GradleWrapperMain.main(GradleWrapperMain.java:61)
Caused by: java.lang.NullPointerException: Cannot get property 'tag' on null obj
ect
        at build_azeclrabgz64sixypldpi3j7.run(C:\2017\ReGameDLL_CS-master\build.
gradle:19)
        at org.gradle.groovy.scripts.internal.DefaultScriptRunnerFactory$ScriptR
unnerImpl.run(DefaultScriptRunnerFactory.java:74)
        ... 38 more


BUILD FAILED

Total time: 59.64 secs

C:\2017\ReGameDLL_CS-master>

C:\2017\ReGameDLL_CS-master>
```

rehlds builds fine, i m using windows 7.

**Comment by justgo97** [comment]
its fixed but now i got another error:

```
C:\2017\ReGameDLL_CS-master>gradlew --max-workers=1 clean buildFixes --stacktrac
e
:buildSrc:compileJava UP-TO-DATE
:buildSrc:compileGroovy UP-TO-DATE
:buildSrc:processResources UP-TO-DATE
:buildSrc:classes UP-TO-DATE
:buildSrc:jar UP-TO-DATE
:buildSrc:assemble UP-TO-DATE
:buildSrc:compileTestJava UP-TO-DATE
:buildSrc:compileTestGroovy UP-TO-DATE
:buildSrc:processTestResources UP-TO-DATE
:buildSrc:testClasses UP-TO-DATE
:buildSrc:test UP-TO-DATE
:buildSrc:check UP-TO-DATE
:buildSrc:build UP-TO-DATE
##teamcity[buildNumber '5.2.0']
:dep/cppunitlite:clean UP-TO-DATE
:regamedll:clean
:regamedll:generateAppVersion
The current ReGameDLL maven version is 5.2.0-dev+m, url: ()
:regamedll:compileReleaseRegamedllFixesRegamedll_mp_gamedllSharedLibraryRegamedl
l_mp_gamedllRegamedll_pch
precompiled.cpp
C:\2017\ReGameDLL_CS-master\regamedll\engine\osconfig.h(36): fatal error C1083:
Cannot open include file: 'stdio.h': No such file or directory
cl : Command line warning D9025 : overriding '/sdl-' with '/GS-'

:regamedll:compileReleaseRegamedllFixesRegamedll_mp_gamedllSharedLibraryRegamedl
l_mp_gamedllRegamedll_pch                                                      F
AILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':regamedll:compileReleaseRegamedllFixesRegamedll_mp_g
amedllSharedLibraryRegamedll_mp_gamedllRegamedll_pch'.
> A build operation failed.
      C++ compiler failed while compiling precompiled.cpp.
  See the complete log at: file:///C:/2017/ReGameDLL_CS-master/regamedll/build/t
mp/compileReleaseRegamedllFixesRegamedll_mp_gamedllSharedLibraryRegamedll_mp_gam
edllRegamedll_pch/output.txt

* Try:
Run with --info or --debug option to get more log output.

* Exception is:
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':regamed
ll:compileReleaseRegamedllFixesRegamedll_mp_gamedllSharedLibraryRegamedll_mp_gam
edllRegamedll_pch'.
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.ex
ecuteActions(ExecuteActionsTaskExecuter.java:69)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.ex
ecute(ExecuteActionsTaskExecuter.java:46)
        at org.gradle.api.internal.tasks.execution.PostExecutionAnalysisTaskExec
uter.execute(PostExecutionAnalysisTaskExecuter.java:35)
        at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.exec
ute(SkipUpToDateTaskExecuter.java:64)
        at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execut
e(ValidatingTaskExecuter.java:58)
        at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecu
ter.execute(SkipEmptySourceFilesTaskExecuter.java:42)
        at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter
.execute(SkipTaskWithNoActionsExecuter.java:52)
        at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execut
e(SkipOnlyIfTaskExecuter.java:53)
        at org.gradle.api.internal.tasks.execution.ExecuteAtMostOnceTaskExecuter
.execute(ExecuteAtMostOnceTaskExecuter.java:43)
        at org.gradle.api.internal.AbstractTask.executeWithoutThrowingTaskFailur
e(AbstractTask.java:310)
        at org.gradle.execution.taskgraph.AbstractTaskPlanExecutor$TaskExecutorW
orker.executeTask(AbstractTaskPlanExecutor.java:79)
        at org.gradle.execution.taskgraph.AbstractTaskPlanExecutor$TaskExecutorW
orker.processTask(AbstractTaskPlanExecutor.java:63)
        at org.gradle.execution.taskgraph.AbstractTaskPlanExecutor$TaskExecutorW
orker.run(AbstractTaskPlanExecutor.java:51)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor.process(Defaul
tTaskPlanExecutor.java:23)
        at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter.execute(Defau
ltTaskGraphExecuter.java:88)
        at org.gradle.execution.SelectedTaskExecutionAction.execute(SelectedTask
ExecutionAction.java:37)
        at org.gradle.execution.DefaultBuildExecuter.execute(DefaultBuildExecute
r.java:62)
        at org.gradle.execution.DefaultBuildExecuter.access$200(DefaultBuildExec
uter.java:23)
        at org.gradle.execution.DefaultBuildExecuter$2.proceed(DefaultBuildExecu
ter.java:68)
        at org.gradle.execution.DryRunBuildExecutionAction.execute(DryRunBuildEx
ecutionAction.java:32)
        at org.gradle.execution.DefaultBuildExecuter.execute(DefaultBuildExecute
r.java:62)
        at org.gradle.execution.DefaultBuildExecuter.execute(DefaultBuildExecute
r.java:55)
        at org.gradle.initialization.DefaultGradleLauncher.doBuildStages(Default
GradleLauncher.java:149)
        at org.gradle.initialization.DefaultGradleLauncher.doBuild(DefaultGradle
Launcher.java:106)
        at org.gradle.initialization.DefaultGradleLauncher.run(DefaultGradleLaun
cher.java:86)
        at org.gradle.launcher.exec.InProcessBuildActionExecuter$DefaultBuildCon
troller.run(InProcessBuildActionExecuter.java:90)
        at org.gradle.tooling.internal.provider.ExecuteBuildActionRunner.run(Exe
cuteBuildActionRunner.java:28)
        at org.gradle.launcher.exec.ChainingBuildActionRunner.run(ChainingBuildA
ctionRunner.java:35)
        at org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProce
ssBuildActionExecuter.java:41)
        at org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProce
ssBuildActionExecuter.java:28)
        at org.gradle.launcher.exec.DaemonUsageSuggestingBuildActionExecuter.exe
cute(DaemonUsageSuggestingBuildActionExecuter.java:50)
        at org.gradle.launcher.exec.DaemonUsageSuggestingBuildActionExecuter.exe
cute(DaemonUsageSuggestingBuildActionExecuter.java:27)
        at org.gradle.launcher.cli.RunBuildAction.run(RunBuildAction.java:40)
        at org.gradle.internal.Actions$RunnableActionAdapter.execute(Actions.jav
a:169)
        at org.gradle.launcher.cli.CommandLineActionFactory$ParseAndBuildAction.
execute(CommandLineActionFactory.java:237)
        at org.gradle.launcher.cli.CommandLineActionFactory$ParseAndBuildAction.
execute(CommandLineActionFactory.java:210)
        at org.gradle.launcher.cli.JavaRuntimeValidationAction.execute(JavaRunti
meValidationAction.java:35)
        at org.gradle.launcher.cli.JavaRuntimeValidationAction.execute(JavaRunti
meValidationAction.java:24)
        at org.gradle.launcher.cli.CommandLineActionFactory$WithLogging.execute(
CommandLineActionFactory.java:206)
        at org.gradle.launcher.cli.CommandLineActionFactory$WithLogging.execute(
CommandLineActionFactory.java:169)
        at org.gradle.launcher.cli.ExceptionReportingAction.execute(ExceptionRep
ortingAction.java:33)
        at org.gradle.launcher.cli.ExceptionReportingAction.execute(ExceptionRep
ortingAction.java:22)
        at org.gradle.launcher.Main.doAction(Main.java:33)
        at org.gradle.launcher.bootstrap.EntryPoint.run(EntryPoint.java:45)
        at org.gradle.launcher.bootstrap.ProcessBootstrap.runNoExit(ProcessBoots
trap.java:54)
        at org.gradle.launcher.bootstrap.ProcessBootstrap.run(ProcessBootstrap.j
ava:35)
        at org.gradle.launcher.GradleMain.main(GradleMain.java:23)
        at org.gradle.wrapper.BootstrapMainStarter.start(BootstrapMainStarter.ja
va:30)
        at org.gradle.wrapper.WrapperExecutor.execute(WrapperExecutor.java:127)
        at org.gradle.wrapper.GradleWrapperMain.main(GradleWrapperMain.java:61)
Caused by: org.gradle.internal.operations.MultipleBuildOperationFailures: A buil
d operation failed.
    C++ compiler failed while compiling precompiled.cpp.
See the complete log at: file:///C:/2017/ReGameDLL_CS-master/regamedll/build/tmp
/compileReleaseRegamedllFixesRegamedll_mp_gamedllSharedLibraryRegamedll_mp_gamed
llRegamedll_pch/output.txt
        at org.gradle.internal.operations.DefaultBuildOperationQueue.waitForComp
letion(DefaultBuildOperationQueue.java:72)
        at org.gradle.nativeplatform.toolchain.internal.NativeCompiler.execute(N
ativeCompiler.java:73)
        at org.gradle.nativeplatform.toolchain.internal.NativeCompiler.execute(N
ativeCompiler.java:39)
        at org.gradle.nativeplatform.toolchain.internal.OutputCleaningCompiler.c
ompileSources(OutputCleaningCompiler.java:46)
        at org.gradle.nativeplatform.toolchain.internal.OutputCleaningCompiler.e
xecute(OutputCleaningCompiler.java:38)
        at org.gradle.nativeplatform.toolchain.internal.OutputCleaningCompiler.e
xecute(OutputCleaningCompiler.java:26)
        at org.gradle.language.nativeplatform.internal.incremental.IncrementalNa
tiveCompiler.doCleanIncrementalCompile(IncrementalNativeCompiler.java:96)
        at org.gradle.language.nativeplatform.internal.incremental.IncrementalNa
tiveCompiler.execute(IncrementalNativeCompiler.java:74)
        at org.gradle.language.nativeplatform.internal.incremental.IncrementalNa
tiveCompiler.execute(IncrementalNativeCompiler.java:37)
        at org.gradle.nativeplatform.internal.BuildOperationLoggingCompilerDecor
ator.execute(BuildOperationLoggingCompilerDecorator.java:38)
        at org.gradle.nativeplatform.internal.BuildOperationLoggingCompilerDecor
ator.execute(BuildOperationLoggingCompilerDecorator.java:22)
        at org.gradle.language.nativeplatform.tasks.AbstractNativeCompileTask.do
Compile(AbstractNativeCompileTask.java:115)
        at org.gradle.language.nativeplatform.tasks.AbstractNativeCompileTask.co
mpile(AbstractNativeCompileTask.java:107)
        at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:75)
        at org.gradle.api.internal.project.taskfactory.AnnotationProcessingTaskF
actory$IncrementalTaskAction.doExecute(AnnotationProcessingTaskFactory.java:243)

        at org.gradle.api.internal.project.taskfactory.AnnotationProcessingTaskF
actory$StandardTaskAction.execute(AnnotationProcessingTaskFactory.java:219)
        at org.gradle.api.internal.project.taskfactory.AnnotationProcessingTaskF
actory$IncrementalTaskAction.execute(AnnotationProcessingTaskFactory.java:230)
        at org.gradle.api.internal.project.taskfactory.AnnotationProcessingTaskF
actory$StandardTaskAction.execute(AnnotationProcessingTaskFactory.java:208)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.ex
ecuteAction(ExecuteActionsTaskExecuter.java:80)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.ex
ecuteActions(ExecuteActionsTaskExecuter.java:61)
        ... 49 more
Caused by: org.gradle.nativeplatform.toolchain.internal.CommandLineToolInvocatio
nFailure: C++ compiler failed while compiling precompiled.cpp.
        at org.gradle.nativeplatform.toolchain.internal.DefaultCommandLineToolIn
vocationWorker.execute(DefaultCommandLineToolInvocationWorker.java:83)
        at org.gradle.nativeplatform.toolchain.internal.DefaultCommandLineToolIn
vocationWorker.execute(DefaultCommandLineToolInvocationWorker.java:30)
        at org.gradle.internal.operations.DefaultBuildOperationQueue$OperationHo
lder.run(DefaultBuildOperationQueue.java:110)
        at org.gradle.internal.concurrent.ExecutorPolicy$PropagateFailures.onExe
cute(ExecutorPolicy.java:76)
        at org.gradle.internal.concurrent.StoppableExecutorImpl$1.run(StoppableE
xecutorImpl.java:40)


BUILD FAILED

Total time: 44.06 secs

C:\2017\ReGameDLL_CS-master>
```

**Comment by justgo97** [comment]
not sure but is this the file the error talking about cause it exists here : 
C:\Program Files\Windows Kits\10\Include\10.0.15063.0\ucrt\stdio.h

**Comment by s1lentq** [comment]
fixed e71deb8

**Comment by justgo97** [comment]
Build succefull :+1:
